config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java

```diff
@@ -1666,6 +1667,57 @@ void strictly_increasing_cluster_state_versions_config_controlled_by_feature_fla
         checkStrictlyIncreasingClusterStateVersionConfig(true, true);
     }
 
+    @Test
+    void testResourceLimitsForSmallNodes() throws Exception {
+        var services = """
+                <services>
+                  <content version="1.0" id="foo">
+                    <redundancy>1</redundancy>
+                    <documents/>
+                    <nodes count='1'>
+                      <resources vcpu='1' memory='3Gb' />
```

Comment:
Consider testing with a 8GB flavor, as that is considered small.

Reply:
👍 Done